libraries/Resample/design_fir.py

```diff
@@ -0,0 +1,76 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.signal as sig
+import os
+
+sr = 1                                 # samplerate
+ripple = {"high": 80, "low": 60}       # stopband attenuation and passband ripple
+cutoff_ny = {"high": 0.9, "low": 0.7}  # cutoff relative to nyquist
+
+fname = "aa_fir.h"
+try:
+    os.remove(fname)
+except FileNotFoundError:
+    pass
+file = open(fname, "a")
+
+file.write("#include <vector>\n\n")
```

Comment:
When generating a header file, I'd suggest using `std::array` for this(as the array lengths are known at compile time and immutable).

Reply:
Would you then recommend to convert the arrays into vectors during load (in `get_fir`)?